autoreject/utils.py

```diff
@@ -405,7 +405,7 @@ def _compute_dots(info, mode='fast'):
     coils = _create_meg_coils(info['chs'], 'normal', info['dev_head_t'],
                               templates)
     my_origin = _check_origin((0., 0., 0.04), info)
-    int_rad, noise, lut_fun, n_fact = _setup_dots(mode, coils, 'meg')
+    int_rad, noise, lut_fun, n_fact = _setup_dots(mode, info, coils, 'meg')
```

Comment:
now the issue is that this will fail for older MNE versions. Could we perhaps monkey-patch `_setup_dots` ?

```py
def _my_setup_dots(mode, info, coils, 'meg'):
    if mne_version < 0.23:
       return _setup_dots(mode, coils, 'meg')
   else:
      return _setup_dots(mode, info, coils, 'meg')
```

Reply:
I tried one in fb484f3